dev/build_release.py

```diff
@@ -196,26 +122,17 @@ def __init__(self, options):
 
       os.environ['NODE_ENV'] = os.environ.get('NODE_ENV', 'dev')
       self.__build_number = options.build_number
-      self.__google_service_account = options.google_service_account
+      self.__gcb_service_account = options.gcb_service_account
       self.__options = options
+      if (self.__options.container_builder
```

Comment:
I removed build_container_image as a flag.
The default container_builder is None. Setting the container builder expresses you want to build a container image. Not setting it expresses you dont so the flag is superfluous. Setting it empty ensure it is off.